packages/template-typescript-minimal/app/pages/home.tsx

```diff
@@ -11,7 +11,7 @@ import HelloTS from '../components/hello-typescript'
 import HelloJS from '../components/hello-javascript'
 
 interface Props {
-    dadJoke: string
+    dadJoke: any
```

Comment:
Why this change?

Reply:
the dadJoke from the api is an object, not a string. More precisely, we can declare its interface. 
```
interface dadJoke {
  id: string,
  joke: string,
  status: number. 
}
```